### ISSUE TITLE
PhantomJS PDF Generation

### ISSUE THREAD

**Issue by CharlesWinter** [vercel/pkg#284]
pkg version : 4.2.5
cmd line: pkg -t node8-linux-x64 package.json --output n

I have an issue using pkg with the reasonably popular npm package [](https://www.npmjs.com/package/html-pdf)

At first I had an error due to the phantomjs binary not being included. 
```
Warning Cannot include file %1 into executable.
  The file must be distributed with executable as %2.
  node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
  path-to-executable/phantom/phantomjs
```

Without the file I would get the following error
`Error: spawn /home/cw/development/myproject/myprojectdir/phantom/phantomjs ENOENT`

However including the proposed binary was enough to get rid of this error, but I am now faced with the following error

`Error: Can't open '/snapshot/myproject/myprojectdir/node_modules/html-pdf/lib/scripts/pdf_a4_portrait.js'`

Can you advise me on how to proceed debugging?

**Comment by dickeylth** [comment]
I'm also using phantomjs and encountered the same error:

```
Warning Cannot include file %1 into executable.
  The file must be distributed with executable as %2.
  node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs
  path-to-executable/phantom/phantomjs
> Warning Cannot include file %1 into executable.
  The file must be distributed with executable as %2.
  node_modules/phantomjs-prebuilt/lib/phantom/bin/phantomjs.exe
  path-to-executable/phantom/phantomjs.exe
```

**Comment by CharlesWinter** [comment]
@dickeylth  To solve your error, you need to include the binary in a folder next to your "compiled" nodejs program called phantom. I ended up solving my problem by using wkhtmltopdf with a system call.

**Comment by dickeylth** [comment]
@CharlesWinter Thank you, I'll get it a try

**Comment by Jucesr** [comment]
I am having the same issue. @CharlesWinter could you provide more detail about your solution

**Comment by CharlesWinter** [comment]
Hi @Jucesr 

I'm struggling to remember exactly what I ended up doing. I seem to remember that in the end I just shoved the binary for wkhtmltopdf https://wkhtmltopdf.org/ in the container I was building. (it was based on alpine so you'll need this https://pkgs.alpinelinux.org/package/edge/testing/x86/wkhtmltopdf if your doing the same.)

I then generated some HTML, saved it in a file, and just called the binary https://stackoverflow.com/questions/20643470/execute-a-command-line-binary-with-node-js

and it worked well, despite being so crude!
